tests/python/common/sampling/test_sampling.py

```diff
@@ -1774,7 +1774,7 @@ def contain_edge(g, sg, u, v):
 
 @pytest.mark.parametrize("dtype", ["int32", "int64"])
 def test_global_uniform_negative_sampling(dtype):
-    warnings.simplefilter("ignore", np.ComplexWarning)
+    # warnings.simplefilter("ignore", np.ComplexWarning)
```

Comment:
Does the following work?
```suggestion
    warnings.simplefilter("ignore", np.exceptions.ComplexWarning)
```

Reply:
you cannot simply comment out this line. please try with above suggestion.